### PR TITLE
scripts: add readme for lib

### PR DESCRIPTION
**Pull request by Halamix2**
**Description**

Changes proposed in this pull request:

-add readme for `prow/scripts/lib` directory
**Related issue(s)**

#2974, #3140

### CODE REVIEW

File: prow/scripts/lib/README.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Cluster
+
+## Overview
+
+The folder contains helper scripts that contains commonly used functions.
```
Comment (superojla) [inline prow/scripts/lib/README.md:5]:
```suggestion
The folder contains helper scripts with commonly used functions.
```

File: prow/scripts/lib/README.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+### Project structure
+
+The structure of the folder looks as follows:
```
Comment (superojla) [inline prow/scripts/lib/README.md:9]:
```suggestion
The folder structure looks as follows:
```

File: prow/scripts/lib/README.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+├── github.sh # This script contains function that configure git.
+├── junit.sh # This script contains functions  used for testing with JUnit.
+├── kyma.sh # This script contains functions used for installing and interfacing with Kyma.
+├── log.sh # This script provides unified logging functions.
+├── testing-helpers.sh # This script contains functions adiding Kyma testing.
+└── utils.sh # This script contains various functions that couldn't be assigned to any of the other helper scripts.
+```
+
+### Example use case
+Following example use case reserves IP address from Google Compute Engine and creates new DNS record that uses the IP address.
```
Comment (superojla) [inline prow/scripts/lib/README.md:27]:
```suggestion
The following example use case reserves an IP address from the Google Compute Engine and creates a new DNS record that uses the IP address.
```

**Review by Ressetkk — APPROVED** [review]

**Review by superojla — APPROVED** [review]
😸

**Pull request merged**